template/module/README.rst

```diff
@@ -37,6 +37,14 @@ Known issues / Roadmap
 
 * ...
 
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/{project_repo}/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback `here <https://github.com/OCA/{project_repo}/issues/new?body=module: {module_name}%0Aversion: {version}%0A%0A**Steps to reproduce**%0A- ...%0A%0A**Current behavior**%0A%0A**Expected behavior**>`_.
```

Comment:
You should add line breaks here (after "feedback"), because it is easy to miss the placeholders as they are easily outside the screen.